extensions-contrib/opentelemetry-emitter/README.md

```diff
@@ -0,0 +1,170 @@
+# OpenTracing Emitter
+
+The OpenTelemetry emitter generates OpenTelemetry Spans for queries.
+
+
+## Configuration
+
+### Enabling
+
+Load the plugin and enable the emitter in `common.runtime.properties`:
+
+Load the plugin:
+
+```
+druid.extensions.loadList=[..., "opentelemetry-emitter"]
+```
+
+Enable the emitter:
+
+```
+druid.emitter=opentelemetry
+```
+
+*_Don't forget to remove the line_ `druid.emitter=noop` _to avoid rewriting_.
+
+## Testing
+###Zipkin
+Run zipkin docker container
+```
+docker run -d -p 9411:9411 openzipkin/zipkin
+```
+Zipkin will be available at `http://localhost:9411`.
+
+Add zipkin dependency to `extensions-contrib/opentelemetry-emitter/pom.xml`:
+```
+<dependency>
+  <groupId>io.opentelemetry</groupId>
+  <artifactId>opentelemetry-exporter-zipkin</artifactId>
+  <version>${your.version}</version>
+</dependency>
```

Comment:
this might be fine for development, but requires recompiling. We should aim to make the extension work out of the box. The most straightforward way to do so is probably to bundle the OTLP exporter by default and let users runs their own otel collector or hook up a tracing vendor which supports OTLP out of the box

Reply:
Yeah, that make sense, I'll remove this section with Zipkin and leave only section with OTLP collector
OTLP exporter is already bundled, so users can use OTLP collector to export traces